### ISSUE TITLE
Bad documentation in: Modelica_LinearSystems2/DiscreteZerosAndPoles.mo

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica_LinearSystems2#35]
**Reported by adrpo on 18 May 2016 04:56 UTC**

The documentation for `Modelica_LinearSystems2.DiscreteZerosAndPoles.fromFile` contains unescaped quotes: `+ "    /dzp.mat "`.

```mo
      annotation (Documentation(info="<html>
<h4>Syntax</h4>
<table>
<tr> <td align=right>  dzp </td><td align=center> =  </td>  <td> DiscreteZerosAndPoles.Import.<b>fromFile</b>(fileName)  </td> </tr>
</table>
<h4>Description</h4>
<p>
Reads and loads a discrete zeros-and-poles transfer function from a mat-file <tt>fileName</tt>. The file must contain the sample time Ts and either the set of variables n1, n2, d1, d2, and k with
the associated first and second order polynomials or the variables p, z, and k with the poles and zeros, written in two column arrays with real and imaginary in the first and
second column respectively. The variable k is the real gain in both cases.


<h4>Example</h4>
<blockquote><pre>


<b>algorithm</b>
  dzp:=Modelica_LinearSystems2.DiscreteZerosAndPoles.Import.fromFile(DataDir + "    /dzp.mat " );
//  zp = (q^2 + 2*q + 3)/(q + 2)/(q^2 + 2*q + 2)
</pre></blockquote>
</html>"));
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1998

**Comment by modelica-trac-importer** [comment]
**Comment by sjoelund&#46;se on 18 May 2016 05:06 UTC**
Note that this is unfortunately valid Modelica code, similar to `info="string"/var "comment string"`

**Comment by tobolar** [comment]
Fixed in commit 648d51be0dde4ed3abe3eb207f833687eb7230b2, see class Modelica_LinearSystems2.DiscreteZerosAndPoles.Import.fromFile in file Modelica_LinearSystems2/DiscreteZerosAndPoles.mo.
